### PR TITLE
Yarn scripts movement

### PR DESCRIPTION
**Pull request by jenniew**
## What changes were proposed in this pull request?
move python package scripts to dist, update python user guide

(Please fill in changes proposed in this patch)

## How was this patch tested?
manual test.

## Related links or issues (optional)
fixed https://github.com/intel-analytics/BigDL/issues/1628

### CODE REVIEW

**Comment by yiheng** [comment]
also remove the python_package folder?

**Comment by jenniew** [comment]
yes, python_package folder is removed.

**Comment by yiheng** [comment]
Can we add the requirements to bigdl python package dependency. So when user pip instal bigdl, all will be installed.

**Comment by yiheng** [comment]
others looks good to me.

**Comment by jenniew** [comment]
This document is for without pip. I think we already put the dependency to pip install setup of bigdl. @zhichao-li , if a user pip install bigdl, would he also installed the dependency?

File: docs/docs/PythonUserGuide/install-without-pip.md [hunk 1]
```diff
@@ -16,6 +16,30 @@
...
+        * Make sure you already install such libraries(python-setuptools, python-dev, gcc, make, zip, pip) for creating virtual environment. If not, please install them first. For example, on Ubuntu, run these commands to install:
+          ```
+            apt-get update
+            apt-get install -y python-setuptools python-dev
+            apt-get install -y gcc make
+            apt-get install -y zip
+            easy_install pip
+          ```	
+         * Create dependency virtualenv package
+            * Under BigDL home directory, you can find ```dist/bin/python_package.sh```. Run this script to create dependency virtual environment package according to dependency descriptions in requirements.txt. You can add your own dependencies in requirements.txt. The current requirements.txt only contains dependencies for BigDL python examples and models.
```
[the remark this answers is not in the document]
Comment (jenniew) [reply docs/docs/PythonUserGuide/install-without-pip.md]: updated

**Comment by hkvision** [comment]
@jenniew Yes. If install from pip, the dependencies, such as `numpy` `pyspark` `six` will automatically be installed.

**Pull request merged**